azure-pipelines.parallel.yml

```diff
@@ -0,0 +1,155 @@
+pr:
+  - master
+
+trigger:
+  - master
+
+variables:
+  - group: fabric-variables
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+jobs:
+  - job: Build
+    steps:
+    # TODO: remove, temporarily disabled to speed debugging
+    # - checkout: none
+    #   clean: false
+
+    # TODO: uncomment, temporarily disabled to speed debugging
+    - template: azure-pipelines.tools.yml
+
+    - script: |
+        yarn
+      displayName: yarn
+
+    - script: |
+        npm run checkchange
+      displayName: check change
+
+    - script: |
+        npm run build
```

Comment:
```suggestion
        yarn build
```

Reply:
Do you know why it was this way on master? Was it purely because of the yarn dep?